docs/src/pages/index.tsx

```diff
@@ -23,57 +37,80 @@ const TempOverride = createGlobalStyle`
     }
 `;
 
-const StyledIndexPage = styled('div')`
-    overflow-y: hidden;
-    overflow-x: hidden;
+interface BreakpointProps {
+    breakpoint?: BreakpointsType;
+}
+
+const StyledIndexPage = styled('div')<BreakpointProps>`
     height: 100vh;
     display: flex;
     flex-direction: column;
-    padding-top: 55rpx;
+    padding-top: 3.4375rem;
+    ${({ breakpoint }) => responsiveCSS(breakpoint, [
+        breakpoints.xs,
+        breakpoints.sm,
+    ])`
+        padding-top: 1rem;
+        min-height: 130vh;
+        margin-bottom: 20rem;
+        display:block;
+    `}
 `;
 
-const StyledIntro = styled('div')`
+const StyledIntro = styled('div')<BreakpointProps>`
     width: 100%;
     max-width: ${CONTENT_WIDTH}px;
     margin: 0 auto;
-    padding-top: 100rpx;
+    padding-top: 6.25rem;
+    ${({ breakpoint }) => responsiveCSS(breakpoint, [
+        breakpoints.xs,
+        breakpoints.sm,
+    ])`
+        padding-top: 1rem;
+    `}
 `;
 
-const StyledLogo = styled('img')`
+interface StyledLogoProps extends BreakpointProps {
+    test?: string;
+}
+
+const StyledLogo = styled('img')<StyledLogoProps>`
     flex: 1 1 auto;
+    box-sizing: border-box;
+    width: 80%;
+    max-width: 22.25rem;
+
+    ${({ breakpoint }) => responsiveCSS(breakpoint, [
+        breakpoints.xs,
+        breakpoints.sm,
+    ])`
+        margin: 2rem auto;
+        position: relative;
+        display: block;
+    `}
 `;
 
-const StyledCaption = styled('p')`
+const StyledCaption = styled('p')<BreakpointProps>`
     padding: 1rem 0;
     font-family: AvenirNext-Medium, Avenir, sans-serif;
```

Comment:
Shouldn't this pull from our theme? This font is slightly different, but maybe it shouldn't be.

Reply:
Think that was an oversight. I'll update it to base.

Reply:
Done.